winston.js

```diff
@@ -0,0 +1,13 @@
+const winston = require('winston');
+
+const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.simple(),
```

Comment:
Let's change this to `winston.format.cli()` so we get some colors :) 
```suggestion
  format: winston.format.cli(),
```